projects/js-packages/charts/src/charts/segmented-bar/stories/index.api.mdx

```diff
@@ -0,0 +1,261 @@
+}`}
+/>
+
+## Exports
+
+<Source
+	language="typescript"
+	code={`// Main exports
+import { SegmentedBar, SegmentedBarUnresponsive } from '@automattic/charts';
+
+// Type exports
+import type {
+	SegmentedBarProps,
+	SegmentedBarSegment,
+	SegmentedBarMarker,
+	SegmentedBarMode,
+} from '@automattic/charts';
+
+// Direct import (tree-shakeable)
+import { SegmentedBar } from '@automattic/charts/segmented-bar';`}
+/>
+
+## Component Variants
+
+### SegmentedBar (Responsive)
+
+The default export that automatically adapts to container width:
+
+<Source
+	language="jsx"
+	code={`import { SegmentedBar } from '@automattic/charts';
+
+<SegmentedBar values={[30, 40, 30]} height={8} />`}
+/>
+
+### SegmentedBarUnresponsive
+
+Fixed-size variant that requires explicit width:
+
+<Source
+	language="jsx"
+	code={`import { SegmentedBarUnresponsive } from '@automattic/charts';
+
+<SegmentedBarUnresponsive values={[30, 40, 30]} width={400} height={8} />`}
+/>
+
+## CSS Classes
+
+The component uses CSS Modules with the following class structure:
+
+| Class | Description |
+|-------|-------------|
+| `.segmentedBar` | Root container element |
+| `.segmentedBar--empty` | Applied when values array is empty |
+| `.segmentedBar__bar` | The horizontal bar container |
+| `.segmentedBar__segment` | Individual segment elements |
+| `.segmentedBar__segment--first` | First segment (has left border radius) |
+| `.segmentedBar__segment--last` | Last segment (has right border radius) |
+| `.segmentedBar__marker` | Position marker element |
+| `.segmentedBar__marker--animated` | Marker with animation enabled |
+| `.segmentedBar__labels` | Labels container |
+| `.segmentedBar__label` | Individual label elements |
+| `.segmentedBar__tooltip` | Tooltip container |
+| `.segmentedBar__tooltipLabel` | Tooltip label text |
+| `.segmentedBar__tooltipValue` | Tooltip value text |
+
+## Display Modes
+
+### Proportional Mode
+
+Segment widths are calculated as a percentage of the total:
+
+```
+Segment width = (segment.value / total) * 100%
+```
+
+Example with values `[25, 75]`:
+- Segment 1: 25%
+- Segment 2: 75%
+
+### Equal Mode
+
+All segments have equal width regardless of values:
+
+```
+Segment width = 100% / number of segments
+```
+
+Example with values `[10, 20, 30]`:
+- All segments: 33.33%
+
+## Marker Positioning
+
+The marker position is calculated as a percentage of the total:
+
+```
+Marker position = (marker.value / total) * 100%
+```
+
+For a bar with values `[25, 50, 25]` (total = 100):
+- `marker.value: 25` → positioned at 25%
+- `marker.value: 75` → positioned at 75%
+
+## Theme Integration
+
+The component uses `GlobalChartsProvider` for theming:
+
+<Source
+	language="jsx"
+	code={`import { GlobalChartsProvider, SegmentedBar } from '@automattic/charts';
+import { jetpackTheme } from '@automattic/charts/providers';
+
+<GlobalChartsProvider theme={jetpackTheme}>
+	<SegmentedBar values={[30, 40, 30]} width={400} />
+</GlobalChartsProvider>`}
+/>
```

Comment:
This section normally goes in the docs, rather than API reference. There's a new section added to the template, since we removed the consumer themes.